modules.d/99memstrack/module-setup.sh

```diff
@@ -20,9 +20,15 @@ install() {
     inst "/bin/memstrack" "/bin/memstrack"
 
     inst "$moddir/memstrack-start.sh" "/bin/memstrack-start"
-    inst_hook cleanup 99 "$moddir/memstrack-report.sh"
-
     inst "$moddir/memstrack.service" "$systemdsystemunitdir/memstrack.service"
-
     $SYSTEMCTL -q --root "$initdir" add-wants initrd.target memstrack.service
+
+    pid=$$
+    if grep -wq "kdumpbase" <<< $(cat /proc/"$pid"/cmdline | tr '\0' ' '); then
```

Comment:
kdumpbase is a out of tree dracut module, it may not be a good idea to be used a keyword here.

Reply:
I didn't find a good way for dracut to determine whether it is making a normal initramfsdisk for 1st kernel, or making a kdump initramfsdisk for 2nd kernel. My original thought was to check the cmdline which invokes dracut. For kdump case, it will have string as "--add 'kdumpbase'", so it can act as a sign. I agree kdumpbase is out of tree for dracut, it may not be suitable for other linux distros other than fedora. Maybe putting the check in kexec-tools side, and patch dracut modules.d/99memstrack/module-setup.sh during its install stage is better?